content/docs/command-reference/remote/modify.md

```diff
@@ -163,7 +163,7 @@ parameters to customize the authentication method:
 - `profile` - credentials profile name to access S3:
 
   ```cli
-  $ dvc remote modify --local myremote profile myprofile
+  $ dvc remote modify --local myremote profile myuser
```

Comment:
let's keep it myprofile, it's not user (otherwise we would call it user)

Reply:
also, can we please avoid touching files that are not related 🙏 :)

Reply:
I would argue that it's related (for consistency) since it's in the same page. Changed to `myprofile` throughout this page and in the GDrive guide now.

> These small things I'll never get be able to get to again -- it's now or never... We keep having this misunderstanding 😢